docs/networking.md

```diff
@@ -0,0 +1,175 @@
+# How to use networking
+
+cloud-hypervisor can emulate one or more virtual network interfaces, represented at the hypervisor host by [tap devices](https://www.kernel.org/doc/Documentation/networking/tuntap.txt"). This guide briefly describes how to setup and use networking with cloud-hypevisor.
+
+## Start cloud-hypervisor with net devices
+
+Use one or more `--net` command-line argument from __cloud-hypervisor__ to specify the emulation of one or more virtual NIC's. The example below instructs cloud-hypervisor to emulate for instance 2 virtual NIC's:
+
+```bash
+./cloud-hypervisor \
+    --cpus 4 \
+    --memory "size=512,file=/dev/shm" \
+    --disk my-root-disk.img \
+    --kernel my-vmlinux.bin \
+    --cmdline "console=ttyS0 reboot=k panic=1 nomodules root=/dev/vda3" \
+    --net tap=ich0,mac=a4:a1:c2:00:00:01,ip=192.168.4.2,mask=255.255.255.0 \
+    --net tap=ich1,mac=a4:a1:c2:00:00:02,ip=10.0.1.2,mask=255.255.255.0
+```
+ 
+The -net argument takes a comma-separated string of key value pairs containing the following 4 keys or fields:
+
+| Name     | Purpose                    | Optional  |
+| -------- |----------------------------| ----------|
+| tap      | tap device name            | Yes       |
+| mac      | vNIC mac address           | Yes       |
+| ip       | tap IP IP address          | yes       |
+| mask     | tap IP netmask             | Yes       |
+
+## Configure the tap devices
+
+After starting cloud-hypervisor as shown above, 2 tap devices with state down will become available at the hypervisor host:
```

Comment:
Almost a nit: I am used to talk about the `host` and not `hypervisor host`.

Reply:
sure. will change this.